property.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS `properties` (
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(255) NOT NULL,
+	`interior` VARCHAR(255) NOT NULL,
+	`furnished` BOOLEAN NOT NULL,
+	`owners` JSON,
+	`garage` BOOLEAN NOT NULL,
+	`coords` JSON NOT NULL,
+	`price` INT NOT NULL,
+	`rent` INT NOT NULL,
+	`rent_date` TIMESTAMP DEFAULT NULL,
+	`stash` TEXT,
+	`outfit` TEXT,
+	`logout` TEXT,
+	`decorations` INT,
+	`garage_slots` JSON,
+	PRIMARY KEY (id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS `property_owners` (
+	`id` INT NOT NULL AUTO_INCREMENT,

```

Comment:
instead of having an id, this table could have the primary key be the combo of property_id and citizenid

Reply:
i think i did the combo thing?